app/src/main/java/com/ryan/heroestopbuilds/MainActivity.java

```diff
@@ -212,18 +252,31 @@ public String onChildPress(String name) {
      * @return final string for DB
      */
     public String prettyPrinter(ArrayList<String> popularSkills) {
-        //add our final list to a new list to be passed to MainActivity
+        ArrayList<String> finalList = new ArrayList<>();
         String lgSpacing = String.format("%" + 3 + "s", "");
         String smSpacing = String.format("%" + 1 + "s", "");
-        ArrayList<String>finalList = new ArrayList<>();
-        for(int i = 0; i <= levelMod; ++i) {
-            if(i <= 2) {
-                finalList.add("Level " + (1 + 3 * i) + ": " + lgSpacing + popularSkills.get(i));
-            } else {
-                finalList.add("Level " + (1 + 3 * i) + ": " + smSpacing + popularSkills.get(i));
+        //Check first skills, this means we got a Chromie
```

Comment:
This block can be simplified below for instance if one of those OR clauses evals correctly have a var be set to 1 and then add it within your string comprehension, would remove the need for an if() else() with mostly identical code

int isChromie = 0;
if(popularSkills.contains("CompoundingAether") || popularSkills.contains("DeepBreath")
               || popularSkills.contains("TimewalkersPursuit") || popularSkills.contains("PeerIntoTheFuture")) {
isChromie =1
}

//for loop
finalList.add("Level " + (isChromie + 3 * i) + ": " + lgSpacing + popularSkills.get(i));